hack/binaries

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+. $(dirname $0)/util
+
+: ${TARGETPLATFORM=}
+: ${CONTINUOUS_INTEGRATION=}
+
+set -ex
+
+progressFlag=""
+if [ "$CONTINUOUS_INTEGRATION" == "true" ]; then progressFlag="--progress=plain"; fi
+
+binariesLegacy() {
+	mkdir -p bin
```

Comment:
indentation seems broken.

I'm not sure whether there is `gofmt`-equivalent for bash, but emacs can indent shell scripts with `C-x h M-x indent`.